### DISCUSSION TITLE
Flight time questions

### DISCUSSION THREAD

**Discussion by kalib-loop** [bitcraze/discussions#391] [Q&A]
Hello, I have been looking at the crazyflies for a little while now and had some questions about flight time of the crazyflies using different decks. with the LED deck and the Loco tracking Deck the crazyflie only gets 3-5 minute flight time. If I want to get that flight time up I assume I can use a larger battery but that comes at the cost of added weight and consequently reduced flight performance. Do you think that adding upgraded motors to the crazyflie as shown in post (https://www.bitcraze.io/2022/10/thrust-upgrade-kit-for-the-crazyflie-2-1/) would help compensate for this issue? Of coarse these motors would come with greater power consumption. My hope is to get the flight time up without a significant reduction in performance. Hopefully that made sense. Also is there a place where I can purchase these motors?

**Comment by tobbeanton** [comment]
Fight time is always a hard question as it depends on so many things. Generally though I think the upgrade kit together with a bigger battery is the best way forward. Even though the thrust upgrade kit reduces the flight time a bit, it will actual be a bit more efficient with added weight. As for the battery we have tested the [Tattu 350mAh](https://www.gensace.de/tattu-350mah-3-7v-30c-1s1p-lipo-battery-pack-with-molex-plug-1-pcs-pack-2170.html) battery with good results. It is a bit thicker though so you would have to use the [long pin headers](https://store.bitcraze.io/collections/spare-parts-crazyflie-2-0/products/male-long-deck-connector) or modify the normal ones. Also notice that the Tattu battery doesn't have a protection circuit so a bit more caution is needed (under charge and short circuit especially).

The thurst upgrade kit should be available in end of November in our shop.

**Reply by kalib-loop** [reply]
Thank you for the quick response. 

Have you tested the upgraded kit with the larger battery?

**Reply by tobbeanton** [reply]
Not thoroughly but my guess is that you would get around 5min flight time with LED-ring and Loco deck.

**Reply by kalib-loop** [reply]
Alright thank you
